backend/capellacollab/projects/toolmodels/modelsources/git/exceptions.py

```diff
@@ -116,7 +116,7 @@ async def github_artifact_expired_handler(
     return await exception_handlers.http_exception_handler(
         request,
         fastapi.HTTPException(
-            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            status_code=status.HTTP_409_CONFLICT,
```

Comment:
For expired artifacts, `410 Gone` should be the better choice. Or alternatively, a 404 because the resource is not / no longer available. I don't see a conflict here.

Reply:
Let's stick with 404 then.